boards/BMS/Inc/App/App_PreChargeSequence.h

```diff
@@ -1,39 +1,82 @@
 #pragma once
 
 #include <stdbool.h>
+#include <stdint.h>
 
-struct PreChargeSequence;
+struct PreCharge;
 
 /**
  * Allocate and initialize a pre-charge sequence
- * @param enable_pre_charge_sequence A function that can be called to enable
+ * @param enable A function that can be called to enable
  * the pre-charge sequence
- * @param disable_pre_charge_sequence A function that can be called to disable
+ * @param disable A function that can be called to disable
  * the pre-charge sequence
+ * @param get_ts_voltage A function that can be called to get the tractive
+ * system voltage in V
+ * @param min_duration_ms The time associated with the
+ * inverter/charger capacitors being charged to 90% of the accumulator voltage
  * @return The created pre-charge sequence, whose ownership is given to the
  * caller
  */
-struct PreChargeSequence *App_PreChargeSequence_Create(
-    void (*enable_pre_charge_sequence)(void),
-    void (*disable_pre_charge_sequence)(void));
+struct PreCharge *App_PreCharge_Create(
+    void (*enable)(void),
+    void (*disable)(void),
+    float (*get_raw_ts_voltage)(void),
```

Comment:
What's the relationship between `get_raw_ts_voltage` and `get_ts_voltage`?

Reply:
The raw ts voltage should be the raw adc converted voltage, while `get_ts_voltage` should be the ts voltage after scaling the ADC value

Reply:
Got it